README.md

```diff
@@ -33,6 +33,7 @@ This plugin can only be used once per Server Block.
 
 ~~~
 records [ZONES...] {
+    $OPTION [OPTION]
```

Comment:
Please make the option syntactically similar to how other plugins implement `fallthrough`.  I.e. just `fallthough`, not  prefixed by `$OPTION`

Reply:
One thought here: `$OPTION` is similar to something like `$INCLUDE` for zone files, using "fallthrough" without some directive to indicate it's an option might conflict with someone who wants to have e.g. `fallthrough.example.org`

Reply:
Ah, yes. The record plugin uses an atypical in-line configuration that makes syntactical parity with other plugins complicated/impossible.

Wouldn't `$OPTION` collide with `$OPTION.example.org.`? (an unusual, yet rfc-valid dns name)

Reply:
It would, but I chose this syntax because it's how other directives like `$TTL` and `$ORIGIN` are expressed in zone files. Notably, this directive is case-sensitive and the parser delimits tokens on spaces afaik, so the user can write out `$option` or `$OPTION.example.com.` to workaround this.

Reply:
Are $TTL and $ORIGIN supported by the records plugin?

Reply:
I would say no, but it appears to be supported by the underlying parser in `dns.NewRR`. See [setup.go:56](https://github.com/coredns/records/blob/a3157e710d9e57c75e4950a3750228f3ed9bb47a/setup.go#L56), where it kind-of cheats by prepending an `$ORIGIN` directive to each record line. The support or lack thereof for these directives isn't mentioned in documentation.

Using the parser from the `file` plugin as you mentioned elsewhere would be one way to support these, and one potential future consequence of this PR would be that such a move would be a little more complicated. However, the support for multiple origins for each records block already starts this plugin down that path.